servicetalk-dns-discovery-netty/src/main/java/io/servicetalk/dns/discovery/netty/DnsServiceDiscovererBuilder.java

```diff
@@ -30,26 +30,54 @@
  */
 public interface DnsServiceDiscovererBuilder {
     /**
-     * The minimum allowed TTL. This will be the minimum poll interval.
+     * Controls min/max TTL values that will influence polling intervals.
+     * <p>
+     * The created {@link ServiceDiscoverer} polls DNS server based on TTL value of the resolved records. Min/max values
+     * help to make sure polling stays within reasonable boundaries. Too frequent DNS queries may generate too much load
+     * for the DNS server, too rare DNS queries may lead to incorrect state if the remote servers changed IPs before
+     * original TTL expired.
+     * <p>
+     * With this overload, there will be no local caching for resolved records.
      *
-     * @param minTTLSeconds The minimum amount of time a cache entry will be considered valid (in seconds).
```

Comment:
Consider describing the default values on each param so that if a user only wants to tweak one off from the default it's easier to find what to put in place.

Reply:
Discussed offline, leaving a comment here for others: we decided not to leak default values in javadoc at least until we are 1.0 to have some flexibility to adjust them.